bin/printSchema.ts

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+import axios from "axios";
+import { readFileSync } from "fs";
+import { buildClientSchema, introspectionQuery, printSchema } from "graphql";
+import process from "process";
+
+import { TestxServer } from "../src";
+
+(async () => {
+  const schemaFile = process.argv[2];
+  const schema = readFileSync(schemaFile, 'utf8');
+
+  const server = new TestxServer(schema);
+  await server.start();
+
+  const response = await axios.post(server.url(), { query: introspectionQuery });
```

Comment:
What about moving this line in a method, we could call it `await server.getGraphqlSchema()`, wdyt?

Reply:
moved